packages/webapp/src/routes/SensorDetailsRoutes.jsx

```diff
@@ -0,0 +1,16 @@
+import { Route } from 'react-router-dom';
+import React from 'react';
+import LocationTasks from '../containers/LocationDetails/LocationTasks';
+import SensorDetail from '../containers/LocationDetails/PointDetails/SensorDetail';
+import UpdateSensor from '../containers/LocationDetails/PointDetails/SensorDetail/EditSensor';
+
+export default function SensorDetailsRoutes() {
+  return (
+    <>
+      {/* <Route path="/sensor/:location_id/tasks" exact component={LocationTasks} /> */}
+      {/* <Route path="/sensor/:location_id/readings" exact component={SensorReadings} /> */}
+      <Route path="/sensor/:location_id/details" exact component={SensorDetail} />
+      <Route path="/sensor/:location_id/edit" exact component={UpdateSensor} />
```

Comment:
The edit page should only be shown to an admin user. I believe you can check this with the `isAdminSelector`. For reference, `GateDetailRoutes.jsx` contains this admin check for the edit page